payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayLauncher.kt

```diff
@@ -84,7 +84,7 @@ class GooglePayLauncher internal constructor(
             PaymentConfiguration.getInstance(activity).publishableKey,
             setOf(PRODUCT_USAGE)
         ),
-        DefaultAnalyticsRequestExecutor()
+        DefaultAnalyticsRequestExecutor(context = activity)
```

Comment:
Should we always try to pass the application context, so that context doesn't get leaked?